client/src/test/java/io/pravega/schemaregistry/client/TestSchemaRegistryClient.java

```diff
@@ -311,7 +312,7 @@ public void testGetEncodingInfo() {
         SerializationFormat serializationFormat = SerializationFormat.custom("custom");
         ByteBuffer schemaData = ByteBuffer.wrap(new byte[0]);
         SchemaInfo schemaInfo = new SchemaInfo("schema1", serializationFormat, schemaData, ImmutableMap.of());
-        String codecType = "gzip";
+        CodecType codecType = new CodecType("gzip");
```

Comment:
It's awkward to have to use a string to denote a CodecType. Can we add some constants for standard codec types like `CodecType.GZIP` and use those here (and other places below).

Reply:
So originally i had an enum for codec types. But since registry service doesnt care about the name of the encoding type, it was recommended that i change it to a String and users could provide mime types which is quite standard way of declaring the encoding types. 

And as for constants strings for names for which we provide the codec implementation (namely gzip and snappy), we have defined constant names which you can see in the Serializers PR.